src/app/pools/components/PoolDeleteForm/PoolDeleteForm.tsx

```diff
@@ -0,0 +1,30 @@
+import { useDispatch, useSelector } from "react-redux";
+import { useNavigate } from "react-router-dom-v5-compat";
+
+import ModelDeleteForm from "app/base/components/ModelDeleteForm";
+import urls from "app/base/urls";
+import { actions as resourcePoolActions } from "app/store/resourcepool";
+import resourcePoolSelectors from "app/store/resourcepool/selectors";
+
+const PoolDeleteForm = ({ id }: { id: number }) => {
```

Comment:
After deleting a new pool, incorrect message is displayed and the side panel is not closed.

![Google Chrome screenshot 001296@2x](https://github.com/canonical/maas-ui/assets/7452681/69aaf26a-3900-4c28-80e8-fa776bf4cd24)

Reply:
Then if you try again, another invalid message is displayed.
![Google Chrome screenshot 001298@2x](https://github.com/canonical/maas-ui/assets/7452681/523af1f4-5dd6-4da4-8550-8c33c0569b7d)